### ISSUE TITLE
Temple Blocks are not warped

### ISSUE THREAD

**Issue by Drullkus** [Chisel-Team/Chisel#132]
![2016-03-17_18 41 24](https://cloud.githubusercontent.com/assets/5010174/13866162/e4f50212-ec6f-11e5-9d38-ccdd3d1180be.png)

[For reference](http://www.eldritchgame.com)

**Comment by tterrag1098** [comment]
Yeah this one is going to be tricky to recreate. Will need a completely custom renderer which we currently have no way of doing. Will be done eventually.

**Comment by ilordnightmarei** [comment]
u dont need render just make a model set for them  woldnt that work beter and have variants posibly

**Comment by tterrag1098** [comment]
That won't work. The model system cannot recreate this effect.

**Comment by Drullkus** [comment]
Bloody heck. Should we just dump them wholesale if players have not already gotten their hands on it?

**Comment by tterrag1098** [comment]
This now works with the `eldritch` render type in CTM, which chisel will soon use. ref https://github.com/Chisel-Team/ConnectedTexturesMod/commit/c927076b510ecbb9f80f13f7bbb3e9c37a084fd9,
